template.yaml

```diff
@@ -153,14 +164,15 @@ Resources:
         LogFormat: !Ref LogFormat
         LogGroup: !Sub "/aws/lambda/CumulusAggProcessUpload-${DeployStage}-${NetworkName}"
       MemorySize: 128
-      Timeout: 800
+      Timeout: 100
```

Comment:
just noticed this as i went along - there's not need for this value to be set so high.